src/components/diagrams/singleLineDiagram/single-line-diagram-pane.js

```diff
@@ -100,8 +102,7 @@ const useDisplayView = (network, studyUuid, currentNode) => {
             function createSubstationSLD(substationId, state) {
                 const substation = network.getSubstation(substationId);
                 if (!substation) return;
-                let name =
-                    useName && substation.name ? substation.name : substationId;
+                let name = getNameOrId(substation?.name, substationId);
```

Comment:
maybe rename the variable ?
`let label = ...` because it could be id or name here...